src/Symfony/Component/Notifier/Bridge/Smsbox/SmsboxOptions.php

```diff
@@ -250,7 +250,6 @@ public static function validateDate(string $date): string
 
     public static function validateDateTime(\DateTime $dateTime)
```

Comment:
we should never use DateTime anymore so we should change all occurences to DateTimeImmutable

Reply:
also, all public static method should be private to me